.pre-commit-config.yaml

```diff
@@ -1,5 +1,5 @@
 repos:
```

Comment:
upgrading this as the previous version there's a bug with error:
`ImportError: cannot import name '_unicodefun' from 'click'`